docs/labels.md

```diff
@@ -0,0 +1,106 @@
+| area/network-policy/scope          | Issues or PRs related to network policy scope (namespace, global, etc.) | Any |
+| area/network-policy/staging        | Issues or PRs related to staging network policies | Any |
+| area/octant                        | Issues or PRs related to Octant | Any |
+| area/ovs/openflow                  | Issues or PRs related to Open vSwitch Open Flow | Any |
+| area/OS/linux                      | Issues or PRs related to the Linux operating system | Any |
+| area/OS/windows                    | Issues or PRs related to the Windows operating system | Any |
+| area/provider/aws                  | Issues or PRs related to aws provider | Any |
+| area/provider/azure                | Issues or PRs related to azure provider | Any |
+| area/provider/gcp                  | Issues or PRs related to gcp provider | Any |
+| area/provider/vmware               | Issues or PRs related to vmware provider | Any |
+| area/routing                       | Issues or PRs related to routing and forwarding | Any |
+| area/security/access-control       | Issues or PRs related to access control | Any |
+| area/security/controlplane         | Issues or PRs related to controlplane security | Any |
+| area/security/dataplane            | Issues or PRs related to dataplane security | Any |
+| area/transit/ip                    | Issues or PRs related to internet protocol version (IP) | Any |
+| area/transit/encapsulation         | Issues or PRs related to encapsulation | Any |
+| area/transit/addressing            | Issues or PRs related to IP addressing category (unicast, multicast, broadcast, anycast) | Any |
+| area/transit/encryption            | Issues or PRs related to transit encryption (IPSec, SSL) | Any |
+| area/transit/qos                   | Issues or PRs related to transit qos or policing | Any |
+| kind/api-change                    | Categorizes issue or PR as related to adding, removing, or otherwise changing an API. | Any |
+| kind/bug                           | Categorizes issue or PR as related to a bug.              | Any                |
+| kind/cleanup                       | Categorizes issue or PR as related to cleaning up code, process, or technical debt | Any |
+| kind/deprecation                   | Categorizes issue or PR as related to feature marked for deprecation | Any |
+| kind/design                        | Categorizes issue or PR as related to design | Any |
+| kind/documentation                 | Categorizes issue or PR as related to a documentation.    | Any                |
+| kind/failing-test                  | Categorizes issue or PR as related to a consistently or frequently failing test | Any |
+| kind/feature                       | Categorizes issue or PR as related to a new feature.      | Any                |
+| kind/support                       | Categorizes issue or PR as related to a support question. | Any |
```

Comment:
does this only apply to issues? This seems to be implied by your other doc.

Reply:
These also apply to PRs. Will update doc to reflect that. A PR that resolves an issue should have the same `kind/<kind>` label.